Source/WebCore/Modules/mediastream/RTCOfferOptions.idl

```diff
@@ -28,4 +28,7 @@
     EnabledBySetting=PeerConnectionEnabled
 ] dictionary RTCOfferOptions : RTCOfferAnswerOptions {
     boolean iceRestart = false;
+
+    [EnabledBySetting=LegacyWebRTCOfferOptionsEnabled] boolean? offerToReceiveVideo;
+    [EnabledBySetting=LegacyWebRTCOfferOptionsEnabled] boolean? offerToReceiveAudio;
```

Comment:
If there are too many concerns about this maybe these can be exposed only for GLib ports....